### PR TITLE
Automatic validation of your citation metadata

### PR DESCRIPTION
**Pull request by fdiblen**
Hello!

We noticed that your repository is using the Citation File Format to store citation metadata in `CITATION.cff`. This Pull Request automates validation of that file using the [cffconvert GitHub Action](https://github.com/marketplace/actions/cffconvert). That way, it's a little bit easier to be robust against future changes to the `CITATION.cff` file.

BTW it's perfectly fine if you don't feel like accepting this Pull Request for whatever reason -- we just thought it might be helpful is all.

We found your repository using a partially automated workflow; if you have any questions about that, feel free to create an issue over at https://github.com/cffbots/filtering/issues/

On behalf of the cffbots team,
@abelsiqueira / @fdiblen / @jspaaks

### CODE REVIEW

File: .github/workflows/cffconvert.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: cffconvert
+
+on:
+  push:
+    paths:
+      - CITATION.cff
```
Comment (cwmeijer) [inline .github/workflows/cffconvert.yml:6]:
```suggestion
  push:
    paths:
      - CITATION.cff
  pull_request:    
    paths:
      - CITATION.cff
```

**Review by cwmeijer — APPROVED** [review]
That looks great! Thanks for the PR. Could you add the trigger on pull requests as well, as suggested? Thanks again!
A+++ PR 👍  🥇

**Comment by fdiblen** [comment]
> That looks great! Thanks for the PR. Could you add the trigger on pull requests as well, as suggested? Thanks again! A+++ PR +1 1st_place_medal

You are very welcome @cwmeijer! I responded to your suggestion.

File: .github/workflows/cffconvert.yml (hunk 1 above)
Comment (cwmeijer) [reply .github/workflows/cffconvert.yml:6]: No, after a bit of thinking I realized your solution is much better. I hadn't noticed the path specification, which is new to me. I'll merge it as proposed :-)

**Pull request merged**